.travis/now

```diff
@@ -60,8 +60,9 @@ _install_coverage () {
 }
 
 _install_rose () {
+    pip install -e .
```

Comment:
@kinow is this a safe/full test, or should we install fully, since that is what the customer will probably do...

Reply:
This is what we do for Cylc so should be fine?

Reply:
Editable install is more likely to get broken. That's why I cjose to build like that in travis. So if not concerned about editable installs failing, you can use the normal approach instead

Reply:
stet